src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/ldap/LdapSessionOptions.Linux.cs

```diff
@@ -11,6 +11,8 @@ public partial class LdapSessionOptions
 
         private bool _secureSocketLayer;
 
+        [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")]
```

Comment:
Let's see what @buyaa-n suggests, but I believe that we shouldn't be adding these as this API is supported in all of our 3 different build flavors. Same comment applies to your changes in the ref and Windows-specific file. The only change I would expect on this PR is the removal of the attribute in the ref file.

Reply:
> The only change I would expect on this PR is the removal of the attribute in the ref file.

Which was my original PR implementation :) I've reverted it back to that...

Reply:
> The only change I would expect on this PR is the removal of the attribute in the ref file.

I suggested to add `[System.Runtime.Versioning.SupportedOSPlatformAttribute("linux")]` instead of removing `[System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")]` though i did not know it was already removed in src, by my understanding it should work like that tagging @terrajobst fr his opinion


It has assembly-level unsupported attributes for `browser;android;ios;tvos` https://github.com/dotnet/runtime/blob/8fdf24b52a875aa3b58180cd23958be4ef796c3d/src/libraries/System.DirectoryServices.Protocols/Directory.Build.props#L10
So if it is supported on all other platforms (like FreeBSD, OSX, etc) I am OK with removing the attribute, especially when the src attribute is already removed in 6.0